### DISCUSSION TITLE
Remaster changes?

### DISCUSSION THREAD

**Discussion by Gnime** [maxiride/pf-encounters#59] [Q&A] [answered]
Hi! I love this tool and have been using it exclusively since it was released. after a brief hiatus i was wondering if this tool is or will be using the Remastered changes?

**Comment by maxiride** [comment]
I know this is coming kinda late, but can you expand about what you are referring to? Are there changes in the rules? Can you link a source on AON?

I am no longer a player of Pathfinder (sigh 😥) so I am a bit out of the loop.

---

I searched a bit and it seems that while the PF2E Remaster did change a lot of things, the creature's level is unchanged (or I missed it). For the sake of the encounter building mechanics only the creature level is needed.

**Comment by Gnime** [comment]
Haha no worries. 

I was just hoping to have the Monster Core creatures added back when i made the post :D 

But if you are no longer playing pathfinder ( i cry) then i get it if you wont update it :)

**Reply by maxiride** [reply, marked as answer]
Even though I am no longer a player I still like to get involved in the Pathfinder world, I can dream 😆, and so I am getting back at releasing some updates on the tool. For instance, I finally fixed the NPC urls that were long broken and I am planning a new creatures update soon with a Remaster toggle to switch back and forth.

**Comment by maxiride** [comment]
🎉 pf2e-encounters v2 is live!

Now legacy and remastered version are available 😃

**Reply by Gnime** [reply]
You absolute beautiful beast! and just as i am finishing maps for a game! Thank you so much for the work dude!
